### ISSUE TITLE
SRW - improve import of JSON with missing data file

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#523]
Currently if I try to import a JSON file with a missing data file (e.g., `*.tif` file), the following error appears in the server's log:
```py
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 602, in process_request_thread
    self.handle_error(request, client_address)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 599, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in execute
    application_iter = app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/beaker/middleware.py", line 155, in __call__
    return self.wrap_app(environ, session_start_response)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 586, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 328, in app_run_simulation
    _start_simulation(data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 816, in _start_simulation
    cfg.job_queue(data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/runner.py", line 200, in __init__
    self.cmd, self.run_dir = simulation_db.prepare_simulation(data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/simulation_db.py", line 437, in prepare_simulation
    template.prepare_aux_files(run_dir, data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 534, in prepare_aux_files
    run_dir,
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 919, in _copy_lib_files
    shutil.copy(str(source_lib.join(f)), str(path))
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/shutil.py", line 119, in copy
    copyfile(src, dst)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/home/vagrant/src/radiasoft/sirepo/run/user/VFiMvPSj/srw/lib/R5.tif'
```
However the front-end does not react to this situation and shows the simulation is running:
![running](https://cloud.githubusercontent.com/assets/13209176/20283835/a40263da-aa88-11e6-8757-1a8f36dc972c.png)

Need to better process this case and throw the error to the front-end to inform user about the missing file.

**Comment by mrakitin** [comment]
The following error is more informative:
```py
  File "/home/vagrant/.pyenv/versions/2.7.10/bin/sirepo", line 9, in <module>
    load_entry_point('sirepo', 'console_scripts', 'sirepo')()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/sirepo_console.py", line 18, in main
    return pkcli.main('sirepo')
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkcli/__init__.py", line 98, in main
    argh.dispatch(parser, argv=argv)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/argh/dispatching.py", line 174, in dispatch
    for line in lines:
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/argh/dispatching.py", line 277, in _execute_command
    for line in result:
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/argh/dispatching.py", line 260, in _call
    result = function(*positional, **keywords)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/pkcli/srw.py", line 42, in run
    _run_srw()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/pkcli/srw.py", line 100, in _run_srw
    main()
  File "<string>", line 214, in main
  File "<string>", line 24, in set_optics
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/srwl_samples.py", line 44, in srwl_opt_setup_transmission_from_image
    d = uti_io.read_image(image_path, show_images=False)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/uti_io.py", line 128, in read_image
    raise ValueError('Provided image file "{}" does not exist.'.format(image_path))
ValueError: Provided image file "R5.tif" does not exist.
```
![error](https://cloud.githubusercontent.com/assets/13209176/20284070/9dfe2c16-aa89-11e6-8813-3c99d9d46723.png)

**Comment by mrakitin** [comment]
Fixed via  		7601942.
